pkgs/applications/networking/instant-messengers/dorion/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, glib-networking
+, gst_all_1
+, libappindicator
+, libayatana-appindicator
+, webkitgtk
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "dorion";
+  version = "1.1.0";
+
+  src = fetchurl {
+    url = "https://github.com/SpikeHD/Dorion/releases/download/v${finalAttrs.version}/Dorion_${finalAttrs.version}_amd64.deb";
+    hash = "sha256-6PKH9OCfQrPc42CTDnwiB3tgd6KqfH/XjinrmfUaqBA=";
+  };
+
+  unpackCmd = ''
+    dpkg-deb -x $curSrc source
+  '';
+
+  runtimeDependencies = [
+    glib-networking
+    libappindicator
+    libayatana-appindicator
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    glib-networking
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    webkitgtk
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/usr/lib/dorion/data/usr
+    mv usr/share/ $out
+    mv usr/{bin,lib} $out/usr/lib/dorion/data/usr
+    ln -s $out/usr/lib/dorion/data/usr/bin/dorion $out/bin/dorion
+
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://github.com/SpikeHD/Dorion";
+    description = "Tiny alternative Discord client";
+    license = lib.licenses.gpl3Only;
+    mainProgram = "dorion";
+    maintainers = with lib.maintainers; [ ];
+    platforms = lib.platforms.linux;
```

Comment:
I dont think this is correct. `lib.platforms.linux` includes things like aarch64-linux but we are only shipping amd64.